src/components/BookCard/index.js

```diff
@@ -20,6 +20,8 @@ import {
   BookLesson,
   UpdateProgress,
   BookLineDivider,
+  BookAuthor,
+  BookLineDividerSmall,
 } from './BookCardElements';
 
 const BookCard = ({ book, removeBookFromStore }) => (
```

Comment:
Styling should be exactly the same as in that Zeplin. Please make sure every style is the same.
Your progress bar:
![progress](https://user-images.githubusercontent.com/35504149/145563054-c586bdd9-e4eb-466e-b20e-ffaf6788b7c4.png)
Zeplin:
<img width="1032" alt="zeplin" src="https://user-images.githubusercontent.com/35504149/145563170-2aff78d5-5c42-4179-ae22-6c399071ad9f.png">

Reply:
Hi @SajjadAhmad14,

Thank you for the feedback. This issue was not raised by the last reviewer?